csrc/scheduler/communication.cpp

```diff
@@ -38,14 +37,6 @@ bool CommunicationScheduler::canScheduleCompileTime(Fusion* fusion) {
     return false;
   }
 
-  if (!HostIrLower::canLower(e)) {
-    scheduler_debug_utils::canScheduleRejectReason(
-        schedulerType(),
-        "Failed to lower the expression to host IR: ",
-        e->toString());
-    return false;
-  }
```

Comment:
If we are removing this here, let's add it at the end of `InsertReshardingPass` to assert all ops have been decomposed to a lowerable state.